### DISCUSSION TITLE
Is it possible to disable polyfilling builtin node modules?

### DISCUSSION THREAD

**Discussion by danielbodart** [parcel-bundler/parcel#5919] [Q&A] [answered]
As documented here: https://v2.parceljs.org/features/node-emulation/#%F0%9F%95%B3%EF%B8%8F-polyfilling-%26-excluding-builtin-node-modules

I'd like to disable ALL polyfilling, I've tried `inlineEnvironment` but I think that only controls the process.env

I've also tried `includeNodeModules: [] ` but that doesn't affect them either. Anything else I can try?

**Comment by mischnic** [comment, marked as answer]
For example in package.json:
```json
{
  "alias": {
    "buffer": false
  }
}
```
should not bundle `buffer` at all.

But I think that only applies to your own project, not npm modules.

> I've tried inlineEnvironment but I think that only controls the process.env

Correct

**Comment by danielbodart** [comment]
Brilliant that works. For anyone else here is the full list to disable:

``` 
"alias": {
    "assert": false,
    "process": false,
    "buffer": false,
    "punycode": false,
    "console": false,
    "querystring": false,
    "constants": false,
    "stream": false,
    "crypto": false,
    "string_decoder" : false,
    "domain": false,
    "sys": false,
    "events": false,
    "timers": false,
    "http": false,
    "tty": false,
    "https": false,
    "url": false,
    "os": false,
    "util": false,
    "path": false,
    "vm": false,
    "zlib": false
  }
  ```
